[DO NOT MERGE] Add warning (#3385)
alphagov/govuk-design-system: src/get-started/index.md

showSubNav: true
---

The GOV.UK Design System is for everyone that works on government services for GOV.UK. It helps digital teams in government make services look like GOV.UK with guides for applying layout, typography, colour and images.

{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}

{% set callout %}

>   Using the GOV.UK Design System for websites that spoof GOV.UK services and information may result in legal action.

Reviewer: ```suggestion
  Using the GOV.UK Design System to imitate government websites and services may result in legal action.
```

My thoughts:

- The word 'spoof' is potentially more readily associated with parodies than scams or malicious deception, though I'm still not sure that 'imitate' is the best word for it. 
- 'government' seems like an appropriately broader net than 'GOV.UK', if just to stop loophole sticklers putting HMRC branding on it instead and saying that gives them a pass. 
- 'websites and services' feels a bit easier for a layperson to understand than 'services and information', IMO.

Of course, happy to be overruled by an actual content person 😛

Author: Those are great, thanks beeps!
